### ISSUE TITLE
undefined urls requested from client code when running  with env:development

### ISSUE THREAD

**Issue by francoisp** [jsbin/jsbin#2816]
On a local install of jsbin. Works fine when NODE_ENV=production but not with development. 
here's the url config:
"url": {
    "host": "10.9.0.42:3000",
    "prefix": "/",
    "ssl": false,
    "static": false,
    "runner": false
  },

The first of 5 errors I get in the chrome debugger:
GET http://10.9.0.42:3000/undefined 404 (Not Found)(anonymous function) @ jquery-1.11.0.min.js:3domManip @ jquery-1.11.0.min.js:3append @ jquery-1.11.0.min.js:3n.fn.(anonymous function) @ jquery-1.11.0.min.js:3Panel @ panel.js:71live @ editors.js:537(anonymous function) @ editors.js:549
jquery-1.11.0.min.js:4 GET http://10.9.0.42:3000/undefined/js/vendor/codemirror5/addon/edit/closebrackets.js?undefined 404 (Not Found)send @ jquery-1.11.0.min.js:4ajax @ jquery-1.11.0.min.js:4load @ addons.js:293loadAddon @ addons.js:397(anonymous function) @ addons.js:411(anonymous function) @ addons.js:430

The output pane shows "this page isnt here." Could I have a missing variable or setting to point to the runner?

Any ideas?

Thanks in advance!
F

**Comment by francoisp** [comment]
I've noticed that by forcing the values for the jsbin that are in the user.js file, I can get the error to go away and the runner does render the code. The save function still does not work.
If I paste this around line 175 in jsbin.js 
 jsbin['root'] = 'http://10.9.0.42:3000';
   jsbin['shareRoot']= 'http://10.9.0.42:3000';
   jsbin['runner']='http://10.9.0.42:3000/runner';
   jsbin['static']= 'http://10.9.0.42:3000';
   jsbin['version']= '3.36.19';

So it looks like the two files fetched from the bin route (user.js and start.js) are not loaded by the browser before the main jsbin.js. I tried to play around with the #if isProduction lines at the bottom of the views/index.html, but no dice.

TIA,
Francois
